.bash_nav

```diff
@@ -7,3 +7,28 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
```

Comment:
The command works by creating a directory in your home called ".short". This will store all of your bookmarks.
Each bookmark is a symlink to your target directory with the specified name.